docs/dev/git.md

```diff
@@ -145,4 +145,4 @@ most intuitive name. [^](#a1)
 
 <b id="f3">[3]</b>: Very, very occasionally (I think this has happened once in
 the history of Synapse), we've had two releases in flight at once. Obviously,
-`release-v1.2.3` is more-stable than `release-v1.3.0`. [^](#a3)
+`release-v1.2` is more-stable than `release-v1.3`. [^](#a3)
```

Comment:
I feel like this example doesn't really ring true anymore?

Reply:
Agreed. I'm inclined to rewrite some other parts of that section (and regenerate the graph, etc.), but I didn't want this PR to fall victim to scope creep. Do you have any preferences for what should be in scope for this PR?

Reply:
No, re-reading it I might rethink this though. With the caveat in the previous sentence of "we've had two release in flight" than it does make sense that `release-v1.2` would be more stable than `release-v1.3`. 

I guess I'm a bit nervous someone will read the second sentence and draw absolute claims...

Reply:
Let's leave well enough alone and get back to this as part of the more general docs overhaul happening as part of the move to mdbook